services/app/src/contexts/ws-context.jsx

```diff
@@ -37,22 +38,12 @@ const WebSocketsContextProvider = ({ children }) => {
     const webSocket = new WebSockets(url);
 
     webSocket.addEventListener('open', () => {
-      console.log('opened');
       setWs(webSocket);
     });
 
-    webSocket.addEventListener('message', (event) => {
-      console.log('message: ', event.data);
-    });
-
-    webSocket.addEventListener('close', () => {
-      console.log('closed');
-    });
-
-    webSocket.addEventListener('error', (err) => {
-      console.log('connection failed');
-      console.error(err);
-    });
+    if(window.location.protocol !== 'https:') {
```

Comment:
Is there any other way of telling apart whether or not we are using develop or prod environment?
We need this logging in non prod environments and do not need it in prod environments.

Maybe process.NODE_ENV would be an option here?